components/ModalInput/styles.ts

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components/native';
+
+import {theme} from '../../utils/theme';
+
+export const ModalView = styled.SafeAreaView`
+  flex: 1;
+  background-color: ${theme.colors.white};
+  padding: 30px 20px 30px 20px;
+`;
+
+export const TextPressable = styled.Text`
+  color: ${theme.colors.blue};
+  font-size: 16px;
+  font-weight: 500;
+`;
+
+export const ViewInput = styled.View`
+  flex: 1;
+  align-items: center;
+  justify-content: center;
+  row-gap: 10px;
+`;
+
+export const TextTitle = styled.Text`
+  font-weight: 800;
+  font-size: 24px;
+  align-self: flex-start;
+  margin-bottom: 10px;
+  color: ${theme.colors.black};
+`;
+
+interface PropsInput {
+  col: boolean;
+}
+
+export const Input = styled.TextInput<PropsInput>`
+  background-color: ${props =>
+    props.col === true ? theme.colors.white : theme.colors.skyBlue};
```

Comment:
props.col === true, if col is already a boolean, there is no need of validation

if prop.col === true, return true, so it's the same to put only prop.col
the same if it's false